scala/dllib/pom.xml

```diff
@@ -193,6 +193,11 @@
             <artifactId>${spark-version.project}</artifactId>
             <version>${project.version}</version>
     	</dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.8.0</version>
```

Comment:
https://github.com/apache/commons-io/tree/rel/commons-io-2.8.0/src/main/java/org/apache/commons/io/serialization
You can copy this hole package(rename the package to com.intel.bigdl...) to bigdl to avoid commons-oi's conflict. And declare this origin site in your code.